libraries/opensk/src/api/crypto/rust_crypto.rs

```diff
@@ -85,6 +85,11 @@ impl ecdh::SecretKey for SoftwareEcdhSecretKey {
         let shared_secret = self.ephemeral_secret.diffie_hellman(&public_key.public_key);
         SoftwareEcdhSharedSecret { shared_secret }
     }
+
+    fn from_slice(_bytes: &[u8; EC_FIELD_SIZE]) -> Option<Self> {
+        // Currently (8/8/2023) cannot create EphemeralSecret from a slice. Update this, if needed.
+        unimplemented!();
```

Comment:
I didn't know that RustCrypto doesn't support this operation. In that case, we probably also shouldn't on this repository.

Reply:
Removed